extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -187,10 +187,14 @@ export class AmpAdUIHandler {
 
   /**
    * @param {string} name
-   * @return {!Element}
+   * @return {?Element}
```

Comment:
Looks like you need to make a runtime assertion for the results of the executions of `addDefaultUiComponent_` since the return type is now nullable.